src/error.rs

```diff
@@ -233,6 +233,19 @@ pub enum AxoassetError {
         details: std::io::Error,
     },
 
+    /// This error indicates that axoasset failed to delete an asset
+    #[error("failed to delete asset at {dest_path}.")]
+    #[diagnostic(help(
+        "Make sure your path is relative to your oranda config or project manifest file."
```

Comment:
is this diagnostic correct (looks like a copy/paste to me but i could be wrong)

Reply:
a lot of the local error diagnostics seem copy pasted so i got scared and just kept on copy pasting :D

Reply:
changed these ones because there is other wording that does make sense